### PR TITLE
feat(ci): automatically convert imports to relative ones

### PR DESCRIPTION
**Pull request by kszucs**
Put up another PR with absolute imports for comparison: #3679

### CODE REVIEW

File: ibis/common/geospatial.py [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 import ibis.expr.datatypes as dt
```
Comment (kszucs) [inline ibis/common/geospatial.py:3]: `absolufy` doesn't seem to handle plain imports, but I guess that's OK.

**Comment by cpcloud** [comment]
@kszucs Can we close this one or #3679?

**Comment by kszucs** [comment]
> @kszucs Can we close this one or #3679?

I put up both so we can decide which one to prefer. Once we have a clear preference I'll close the other.

File: ibis/common/geospatial.py (hunk 1 above)
Comment (MarcoGorelli) [reply ibis/common/geospatial.py:3]: Hey, thanks for using `absolufy-imports` - I don't think plain imports can be written as relative, can they?

Comment (kszucs) [reply ibis/common/geospatial.py:3]: Hi @MarcoGorelli! Thanks for chiming in!

It should be possible to rewrite them an relative, in this case `from ..expr import datatypes as dt`

Comment (MarcoGorelli) [reply ibis/common/geospatial.py:3]: true, but that would change it from a direct import to a from import, not sure if people would want that...worth considering anyway, thanks!

**Comment by gforsyth** [comment]
> I put up both so we can decide which one to prefer. Once we have a clear preference I'll close the other.

I don't have a strong opinion one way or the other re: absolute vs. relative, but I'm definitely on board with standardizing on one or the other.

Given that `absolufy` currently doesn't convert plain imports to relative `from` imports, it seems like we'd get more consistency from standardizing on absolute imports.  So if it just comes down to "pick one", I think I'd lean towards absolutes.

**Comment by cpcloud** [comment]
I would agree with @gforsyth. Let's stick with absolute.

**Comment by cpcloud** [comment]
@kszucs Should we close this out and go with #3679?

**Comment by kszucs** [comment]
Closing in favor of #3679

**Pull request closed without merging**